src/home/index.js

```diff
@@ -0,0 +1,12 @@
+import './make-user.js';
+import api from '../services/api.js';
+
+const userSignUp = document.getElementById('form');
+
+userSignUp.addEventListener('play', (event) => {
+    event.preventDefault();
+    const formData = new FormData(userSignUp);
+    const user = user(formData);
```

Comment:
If you followed the previous comments, ( you dont have too follow them they are just suggestions), this like should look like this. 
`    const user = makeUser(formData);
`
To reference that makeUser function we imported at the top.